README.md

```diff
@@ -46,7 +46,7 @@ images as lockscreen background depending on argument provided by user.
 
 > Note: Make sure your system has all dependencies satisfied
 
-- [i3lock-color](https://github.com/PandorasFox/i3lock-color) - i3lock fork with additional features( >= 2.11-c )
+- [i3lock-color](https://github.com/Raymo111/i3lock-color) - i3lock fork with additional features(`>= 2.13.c.3`)
 - [imagemagick](https://www.imagemagick.org/script/index.php) - To apply effects to images
 - [xdpyinfo](https://www.x.org/archive/X11R7.7/doc/man/man1/xdpyinfo.1.xhtml), [xrandr](https://www.x.org/wiki/Projects/XRandR/), [bc](https://www.gnu.org/software/bc/) and [feh](https://feh.finalrewind.org/) - To find screen resolution, set custom blur level and wallpaper handling.
```

Comment:
What I found used not mentioned (possibly to basic except xrdb)

/nix/store/3qkar099302spwvqznyljf8qv1gc0mvi-xset-1.2.4/bin/xset
/nix/store/s7crpcbda751bx87jyrf989ln8l6vbg3-gnused-4.8/bin/sed
/nix/store/0i6vphc3vnr8mg0gxjr61564hnp0s2md-gnugrep-3.6/bin/grep
/nix/store/nzqd1chpvn04xm41944gs7265vam6jhh-xrdb-1.2.0/bin/xrdb

Improvement: dunstctl support (optional)

Reply:
This dependency list is messy in general. Can we just list each dependency, one per line, without explanations...

- i3lock-color (>= 2.13.c.3)
- dep1
- dep2
- someotherdep (optional)